manifests/Microsoft/NuGet/5.5.1.yaml

```diff
@@ -0,0 +1,12 @@
+Id: Microsoft.NuGet
+Version: 5.5.1
+Name: NuGet
+Publisher: Microsoft Corporation
+Homepage: https://www.nuget.org/
+License: © Microsoft Corporation. All rights reserved.
+Description: NuGet Command Line
+Installers: 
+    - Arch: x86
+      Url: https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
```

Comment:
This is unfortunately a fragile implementation. 

See https://docs.microsoft.com/en-us/nuget/api/tools-json for details. 

The latest link will point to a new version soon enough.